testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/harness.js

```diff
@@ -57,6 +57,19 @@ NX.TestHarness.start(
     },
 
     {
+      group: 'Search',
+      items: [
+        {
+          group: 'FT',
+          items: [
+            { url: 'testsuite-oss/search/search_basic.t.js' }
```

Comment:
this test will never pass in the browser due to the dependency on Java code, correct?